indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisor.java

```diff
@@ -3339,43 +3346,43 @@ private void checkTaskDuration() throws ExecutionException, InterruptedException
     final AtomicInteger numStoppedTasks = new AtomicInteger();
     // Sort task groups by start time to prioritize early termination of earlier groups, then iterate for processing
     activelyReadingTaskGroups.entrySet().stream().sorted(
-            Comparator.comparingLong(
```

Comment:
Please revert all the formatting changes in this file.

Reply:
Done.